lib/efi_event.h

```diff
@@ -48,6 +48,18 @@
 #define PACKED __attribute__((__packed__))
 #endif
 
+/* GUIDs used by UEFI/GPT are stored in a mixed-endian layout, where the first
+ * three fields are serialized as little-endian and the final two fields are
+ * serialized as big-endian. This differs from uuid_t which is entirely a
+ * big-endian layout (just an array of 16 bytes).
+ */
```

Comment:
That is an interesting arrangement of structure members in UEFI/GPT. If we are not using the uuid library we may have to revision and track this structure. @williamcroberts your thoughts?

Reply:
Do we have a way to revision and track all the other UEFI structures? Given that this definition is from the spec, if we do revision tracking, we should track all of the UEFI-spec structures together.

Reply:
True, we don't do it as yet.